context.go

```diff
@@ -139,12 +143,46 @@ type Mount struct {
 }
 
 type Docker struct {
-	Name            string
-	NumContainers   int
-	NumImages       int
-	Version         string
-	ApiVersion      string
-	GoVersion       string
-	OperatingSystem string
-	Architecture    string
+	Name               string
+	NumContainers      int
+	NumImages          int
+	Version            string
+	ApiVersion         string
+	GoVersion          string
+	OperatingSystem    string
+	Architecture       string
+	CurrentContainerID string
+}
+
+func GetCurrentContainerID() string {
+	file, err := os.Open("/proc/self/cgroup")
+
+	if err != nil {
+		if os.IsExist(err) {
+			log.Printf("Fail to open /proc/self/cgroup: %s\n", err)
```

Comment:
This log line is going to make `docker-gen` noisy when it runs outside a container. I don't think it's necessary.